docs/subnets/network-runner.md

```diff
@@ -124,13 +124,25 @@ Additional optional parameters which can be passed to the start command:
 
 ```bash
   --plugin-dir ${AVALANCHEGO_PLUGIN_PATH} \
-	--custom-vms '{"subnetevm":"/tmp/subnet-evm.genesis.json"}'
-	--global-node-config '{"index-enabled":false, "api-admin-enabled":true,"network-peer-list-gossip-frequency":"300ms"}'
+  --blockchain-specs '[{"vm_name":"subnetevm","genesis":"/tmp/subnet-evm.genesis.json"}]' \
+	--global-node-config '{"index-enabled":false, "api-admin-enabled":true,"network-peer-list-gossip-frequency":"300ms"}' \
 	--custom-node-configs" '{"node1":{"log-level":"debug","api-admin-enabled":false},"node2":{...},...}'
 ```
 
-`--plugin-dir` and `--custom-vms` are parameters relevant to Subnet operation.
-See the [Subnet section](#network-runner-rpc-server-subnet-evm-example) for details about how to run Subnets.
+`--plugin-dir` and `--blockchain-specs` are parameters relevant to Subnet operation.
+
+`--plugin-dir` can be used to indicate to ANR where it will find plugin binaries for your own VMs. It is optional. If not set, ANR will assume a default location which is relative to the `avalanchego-path` given.
+
+`--blockchain-specs` specifies details about how to create your own blockchains. It takes a JSON array for each blockchain, with the following possible fields:
+
+```
+   "vm_name": human readable name for the VM
+   "genesis": path to a file containing the genesis for your blockchain (must be a valid path)
+   **optional** subnet_id: valid ID of a subnet you want the blockchain to be deployed to
```

Comment:
I should avoid telling about the subnet_id parameter here, because start rejects it

Reply:
ok